src/query/storages/fuse/src/metrics/fuse_metrics.rs

```diff
@@ -95,17 +95,17 @@ pub fn metrics_inc_block_index_write_milliseconds(c: u64) {
     increment_gauge!(key!("block_index_write_milliseconds"), c as f64);
 }
 
-pub fn metrics_inc_block_index_read_nums(c: u64) {
-    increment_gauge!(key!("block_index_read_nums"), c as f64);
-}
+// pub fn metrics_inc_block_index_read_nums(c: u64) {
```

Comment:
No used anymore?

Reply:
Others LGTM

Reply:
> No used anymore?

yes, replaced by `metrics_inc_cache_access_count` in the `CachedReader`. and the label of it changed to `bloom_index_filter_cache_access_count`